### DISCUSSION TITLE
Catch ruleset in osu! triangles doesn't work okay (Visual category)

### DISCUSSION THREAD

**Discussion by greenmg2003** [ppy/osu#16925] [Q&A] [answered]
Just wondering why I feel like catcher in osu! triangles skin functions not so well. Especially the lightning when the catcher catches the fruit. It's just too bright/too large that I may probably can't look well to catch the drop streams or fruit streams. Should this be fixed?
![osu_2022-02-20_11-04-36](https://user-images.githubusercontent.com/65149680/154828012-8df45b9b-36ef-4e85-8093-679fefed8abd.jpg)

**Comment by bdach** [comment, marked as answer]
Has been reported before (https://github.com/ppy/osu/issues/9515), for now you can disable explosions via game settings.

Default skin is pending a redesign anyhow, although that's not likely to be one that will happen anytime soon.

**Reply by greenmg2003** [reply]
Thanks for letting me know! I'll check it soon!
Edit: I checked and it worked better :>
